README.md

```diff
@@ -98,6 +98,8 @@ Beautifier Options:
   -k, --keep-array-indentation      Preserve array indentation
   -x, --unescape-strings            Decode printable characters encoded in xNN notation
   -w, --wrap-line-length            Wrap lines at next opportunity after N characters [0]
+  -a, --wrap-attributes             Wrap attributes to new lines
+  -i, --wrap-indent                 Indent wrapped attributes to after N characters [indent-size]  
```

Comment:
Let's change the name of this to `wrap-attributes-indent-size`.  It makes it clean the the indent we're talking about is specific to the attributes.